presto-benchmark/src/main/java/com/facebook/presto/benchmark/AbstractOperatorBenchmark.java

```diff
@@ -225,7 +225,8 @@ protected final OperatorFactory createHashProjectOperator(int operatorId, PlanNo
         RowExpression translatedHashExpression = translate(hashExpression.get(), variableToInputMapping.build());
 
         PageFunctionCompiler functionCompiler = new PageFunctionCompiler(localQueryRunner.getMetadata(), 0);
-        projections.add(new PageProjectionWithOutputs(functionCompiler.compileProjection(session.getSqlFunctionProperties(), translatedHashExpression, Optional.empty()).get(), new int[] {types.size()}));
+        projections.add(new PageProjectionWithOutputs(functionCompiler.compileProjection(session.getSqlFunctionProperties(), translatedHashExpression, Optional.empty()).get(), new int[] {
```

Comment:
seems like an accidental change

Reply:
It was auto fixed by checkstyle due to changing this file on save since line is too long.